src/components/TodoForm.vue

```diff
@@ -1,80 +1,111 @@
 <template>
-  <div class="w-full h-full search-bar md:desktop-search-bar">
-    <div
-      v-if="popup"
-      class="z-50 absolute flex justify-center items-center inset-0 w-full h-full"
-    >
-      <PopUp
-        @removeTodo="removeTodo"
-        @togglePopup="togglePopup"
-        :index="selectedTaskIndex"
-      />
-    </div>
-    <div class="todo-list">
-      <Header @todoAdded="addTodo" />
-      <div class="flex flex-col items-center gap-8 md:gap-12">
-        <Search v-model="handleSearch" v-if="todos.length" />
-        <Sorting :todos="todos" />
-        <SingleToDo
-          :todos="searchTodos"
+  <div
+    class="w-screen h-screen flex justify-center"
+    @click="handleClickOutside"
+  >
+    <div class="to-do-form w-[287px] h-[68px] h md:w-[600px]">
+      <div
+        v-if="popup"
```

Comment:
If this is a boolean please change the name to something more descriptive and use is prefix